### PR TITLE
MSAL package

### PR DESCRIPTION
**Pull request by roederja2**
Checklist

- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [ ] Package does not vend other packages

It actually installs "tests" as a top level module. I have an outstanding PR to fix this upstream. Do we have to wait until the fix gets released or can we do something in the meantime?

- [X] Build number is 0
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/msal/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - requests >=2.0.0,<3
+    - pyjwt[crypto] >=1.0.0,<2
```
Comment (scopatz) [inline recipes/msal/meta.yaml]: You cannot use square brackets in dependendcies

Comment (roederja2) [reply recipes/msal/meta.yaml]: It looks weird to me as well, but that comes straight from the source package and it seems to work fine...

Comment (scopatz) [reply recipes/msal/meta.yaml]: What I am saying is that it won't work properly for conda. That is a method pip has for specifying optional dependendcies. It doesn't match the conda package spec, and so won't install those dependendcies correctly. It isn't possible to just copy-paste the dependendcy list from setup.py.

Comment (roederja2) [reply recipes/msal/meta.yaml]: Ok, I removed it. The pyjwt conda package seems to depend on `cryptography` anyway. The [crypto] bit just seems to be ignored by conda though. conda install pyjwt[crypto] is the same as conda install pyjwt.

**Pull request merged**